WordPress/Classes/ViewRelated/Jetpack/Branding/Fullscreen Overlay/JetpackFullscreenOverlayGeneralViewModel.swift

```diff
@@ -369,39 +397,46 @@ private extension JetpackFullscreenOverlayGeneralViewModel {
+                                                    comment: "Title of a screen that prompts the user to switch the Jetpack app.")
 
             static let generalContinueButtonTitle = NSLocalizedString("jetpack.fullscreen.overlay.phaseFour.general.continue.title",
-                                                                            value: "Do this later",
-                                                                            comment: "Title of a button that dismisses an overlay that prompts the user to switch the Jetpack app.")
+                                                                      value: "Do this later",
+                                                                      comment: "Title of a button that dismisses an overlay that prompts the user to switch the Jetpack app.")
         }
 
         enum NewUsers {
             static let generalTitle = NSLocalizedString("jetpack.fullscreen.overlay.newUsers.title",
-                                                                  value: "Give WordPress a boost with Jetpack",
-                                                                  comment: "Title of a screen that prompts the user to switch the Jetpack app.")
+                                                        value: "Give WordPress a boost with Jetpack",
+                                                        comment: "Title of a screen that prompts the user to switch the Jetpack app.")
 
             static let subtitle = NSLocalizedString("jetpack.fullscreen.overlay.newUsers.subtitle",
-                                                                  value: "Jetpack lets you do more with your WordPress site. Switching is free and only takes a minute.",
-                                                                  comment: "Title of a screen that prompts the user to switch the Jetpack app.")
+                                                    value: "Jetpack lets you do more with your WordPress site. Switching is free and only takes a minute.",
+                                                    comment: "Title of a screen that prompts the user to switch the Jetpack app.")
+        }
+
+        enum SelfHosted {
+            static let generalTitle = NSLocalizedString("jetpack.fullscreen.overlay.selfHosted.title",
+                                                        value: "Your site has the Jetpack plugin",
+                                                        comment: "Title of a screen that prompts the user to switch the Jetpack app.")
+
+            static let subtitle = NSLocalizedString("jetpack.fullscreen.overlay.selfHosted.subtitle",
+                                                    value: "The Jetpack mobile app is designed to work in companion with the Jetpack plugin. Switch now to get access to stats, notifications, reader, and more.",
```

Comment:
To be consistent with the copy for other phases
```suggestion
                                                    value: "The Jetpack mobile app is designed to work in companion with the Jetpack plugin. Switch now to get access to Stats, Reader, Notifications and more.",
```